src/credentials/FabricTable.cpp

```diff
@@ -95,7 +95,7 @@ CHIP_ERROR FabricInfo::CommitToStorage(PersistentStorageDelegate * storage)
 
     {
         Crypto::P256SerializedKeypair serializedOpKey;
-        if (mOperationalKey != nullptr)
+        if (!mHasExternallyOwnedOperationalKey && mOperationalKey != nullptr)
```

Comment:
But if `mHasExternallyOwnedOperationalKey` why are we wasting time creating a random keypair and setting up `serializedOpKey` if it will be ignored?  Seems like we should have something more like:
```suggestion
        if (!mHasExternallyOwnedOperationalKey) {
            if (mOperationalKey != nullptr) {
```
```
...
            } 
            else 
            {
                // random keypair bits here
            }
        }
```

Reply:
Will do